lib/matplotlib/backends/backend_wx.py

```diff
@@ -1090,15 +1087,17 @@ def _refresh_rubberband(self, previous_rubberband=None):
 class FigureCanvasWx(_FigureCanvasWxBase):
     # Rendering to a Wx canvas using the deprecated Wx renderer.
 
-    def draw(self):
-        """
-        Render the figure using RendererWx instance renderer, or using a
-        previously defined renderer if none is specified.
-        """
-        DEBUG_MSG("draw()", 1, self)
+    def _draw(self):
+        DEBUG_MSG("_draw()", 1, self)
```

Comment:
just kill these DEBUG_MSGs?

Reply:
Thanks. Will do so.

Reply:
We can always remove them later (or now if you want), the point was just not to introduce more of it.

Reply:
And/or replace by logging messages.

Reply:
Maybe combine `_draw()` and `draw()` by adding a `refresh=True` argument to `draw()` that controls whether `self.Refresh()` is called, and then using `self.draw(refresh=False)` instead of `self._draw()`.